docs/zkapps/tutorials/11-advanced-account-updates.mdx

```diff
@@ -13,7 +13,7 @@ zkApps can now be deployed to Berkeley Testnet.
 
 :::note
 
-This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.9.6.
+This tutorial was last tested with [o1js](https://www.npmjs.com/package/o1js) 0.12.1.
```

Comment:
if you have tested with 12.1, great! We have not officially tested with o1js, so the SnarkyJS release is accurate